apps/wallet/src/ui/app/pages/site-connect/index.tsx

```diff
@@ -141,12 +141,7 @@ function SiteConnectPage() {
 								<div className="flex flex-col gap-3">
 									<SectionHeader title="Locked & Unavailable" />
 									{lockedAccounts?.map((account) => (
-										<AccountListItem
-											key={account.id}
-											isLocked
-											selected={false}
-											address={account.address}
-										/>
+										<AccountListItem key={account.id} selected={false} account={account} />
```

Comment:
Not for this but at line 134 we should change `accounts` to `unlockedAccounts`

Reply:
updated